### PR TITLE
add serial cmd: clock

### PR DESCRIPTION
**Pull request by GitMeGet**
- type help to see usage
- help message formatting is a bit off
- bit hard to see what you're typing when the prints are happening in the main loop

### CODE REVIEW

**Comment by jamesi8086** [comment]
please update the code style to be compliant with the rest of the project

File: alpha/serial_cmd.cpp [hunk 1]
```diff
@@ -164,12 +164,68 @@ static void register_version()
   ESP_ERROR_CHECK( esp_console_cmd_register(&cmd) );
 }
 
+static struct {
```
Comment (jamesi8086) [inline alpha/serial_cmd.cpp]: code style: braces on new line
{
}

File: alpha/serial_cmd.cpp [hunk 2]
```diff
@@ -164,12 +164,68 @@ static void register_version()
...
+
 //
 // Register command callbacks
 // - this has to be at the end after all static functions
 //
 void _TS_SerialCmd::register_commands()
 {
   register_version();
+  register_clock_cmd();
+  esp_console_register_help_command();
```
Comment (jamesi8086) [inline alpha/serial_cmd.cpp:237]: register_help_cmd ?

**Review by jamesi8086 — APPROVED** [review]
some changes like style and naming

**Comment by jamesi8086** [comment]
good point , might need to temporarily disable debug in runtime when typing is in progress

**Comment by jamesi8086** [comment]
feel free to create a ticket to improve on that

File: alpha/serial_cmd.cpp (hunk 2 above)
Comment (GitMeGet) [reply alpha/serial_cmd.cpp:237]: yup this gives you the "help" command, their API

**Review by jamesi8086 — APPROVED** [review]

**Pull request merged**